src/components/daily/physicaltooltip/PhysicalTooltip.js

```diff
@@ -57,17 +58,29 @@ class PhysicalTooltip extends React.Component {
       <div key={'physical'} className={styles.pa}>
         <div className={styles.label}>{t('Intensity')}</div>
         <div className={styles.value}>
-          {t(`${this.props.physicalActivity.reportedIntensity}-pa`)}
+          {t(`${pa.reportedIntensity}-pa`)}
         </div>
       </div>,
       <div key={'duration'} className={styles.pa}>
         <div className={styles.label}>{t('Duration')}</div>
-        <div className={styles.value}>{`${d.value}`}</div>
-        <div className={styles.units}>{`${d.units}`}</div>
+        <div className={styles.value}>{`${d.value} ${d.units}`}</div>
       </div>,
     ];
+    const inputTime = [];
+    if (pa.inputTime) {
+      inputTime.push(
+        <div key={'inputTime'} className={styles.pa}>
+          <div className={styles.label}>
+            {t('Entered at')}
+          </div>
+          <div className={styles.value}>
+            {t('{{date}}', {date: formatInputTime(pa.inputTime, timePrefs)})}
```

Comment:
We will have a translation string `{{date}}` with nothing to be translated.
```suggestion
            {formatInputTime(pa.inputTime, timePrefs)}
```